extensions/amp-story/amp-story.md

```diff
@@ -331,6 +335,26 @@ A "poster" is an image that displays in the UI until your story is loaded. The p
 
 The `<amp-story>` component contains one or more [`<amp-story-page>`](#pages:-amp-story-page) components, containing each of the individual screens of the story.  The first page specified in the document order is the first page shown in the story.
 
+### Landscape orientation and full bleed desktop experience opt in
+
+If the `supports-landscape` attribute is specified on the `<amp-story>` element, it will:
+
+  * Allow the story to be seen when a mobile device is held in a landscape orientation.
+  * Change the desktop experience to an immersive full bleed mode, replacing the default three portrait panels experience.
+
+Usage: `<amp-story ... supports-landscape>...</amp-story>`
+
+<figure class="centered-fig">
+  <span class="special-char">Before:</span>
+  <amp-anim alt="Desktop three panels experience" layout="flex-item" src="https://raw.githubusercontent.com/ampproject/amphtml/master/extensions/amp-story/img/amp-story-desktop-three-panels.gif" width="400" height="299">
+  <noscript><img width="400" src="https://raw.githubusercontent.com/ampproject/amphtml/master/extensions/amp-story/img/lamp-story-desktop-three-panels.gif" /></noscript>
+  </amp-anim>
+  <span class="special-char">After:</span>
+  <amp-anim alt="Desktop full bleed experience" layout="flex-item" src="https://raw.githubusercontent.com/ampproject/amphtml/master/extensions/amp-story/img/amp-story-desktop-full-bleed.gif" width="400" height="299">
+  <noscript><img width="400" src="https://raw.githubusercontent.com/ampproject/amphtml/master/extensions/amp-story/img/amp-story-desktop-full-bleed.gif" /></noscript>
+  </amp-anim>
+</figure>
```

Comment:
Updated these images but they still don't appear to be working - @Enriqe & @newmuis do you know if this is because the files haven't been approved yet?

Reply:
I think last time I tried doing this they wouldn't appear until I merged the PR.

Reply:
I think last time I tried this they actually showed up when I clicked on "View file".

@gmajoulet are you able to see the images?

Reply:
thanks!